Osmosis External Incentives

youngjoon-lee opened the pull request:
References:
https://agora.terra.money/t/proposal-luna-incentives-for-osmosis-liquidity-pools/2244
https://forum.scrt.network/t/proposal-scrt-on-osmosis-early-pool-incentives-to-mitigate-slippage-and-boost-liquidity/4737
https://www.mintscan.io/osmosis/proposals/62
https://www.mintscan.io/osmosis/proposals/77
https://commonwealth.im/osmosis/proposal/discussion/2727-match-external-incentives-for-scrtatom-and-scrtosmo-pairs

Commit:
Osmosis External Incentives

gyuguen reviewed (approved):
LGTM 😀

It would be nice to hear other people's opinions.

Commit:
add a link

Commit:
typo

Commit:
more accurate

allenwkkho committed:
Update README.md

Checked and Edited!

Commit:
update MED amounts

Commit:
add json

youngjoon-lee commented:
Since a day has passed, I updated MED amounts based on the new average 7-day closing price.

Also, added a JSON for the on-chain gov which looks like below after it's submitted:
![image](https://user-images.githubusercontent.com/5462944/143605382-ffccfcfc-13a8-47b4-b207-71582ae81d08.png)

youngjoon-lee commented:
If there's no change needed, I'll post this on the Osmosis Commonwealth tomorrow (11/27 Sat).
